README.md

```diff
@@ -19,10 +19,10 @@ pod 'ComponentKit', '~> 0.9'
 
 ## Contributing
 
-See the CONTRIBUTING file for how to help out.
+See the [CONTRIBUTING](https://github.com/facebook/componentkit/blob/master/CONTRIBUTING.md) file for how to help out.
```

Comment:
@modocache thanks! Pull requests where you get to learn new things are the best kind of pull requests :smile: